test/Microsoft.IdentityModel.Tokens.Tests/IdentityComparerTests.cs

```diff
@@ -572,11 +572,11 @@ public void CompareSaml2SecurityTokenHandlers()
         {
             TestUtilities.WriteHeader($"{this}.CompareSaml2SecurityTokenHandlers", true);
             var context = new CompareContext($"{this}.CompareSaml2SecurityTokenHandlers");
-            var saml2SecurityTokenHandler1 = new Saml2SecurityTokenHandler { Serializer = null };
+            var saml2SecurityTokenHandler1 = new Saml2SecurityTokenHandler { Serializer = new Saml2Serializer() };
             var saml2SecurityTokenHandler2 = new Saml2SecurityTokenHandler();
             IdentityComparer.AreEqual(saml2SecurityTokenHandler1, saml2SecurityTokenHandler2, context);
 
-            Assert.True(context.Diffs.Count(s => s == "Serializer:") == 1);
+            Assert.True(context.Diffs.Count(s => s == "Serializer:") == 0);
```

Comment:
Do we need this check?

Reply:
No, I don't think we do. I wanted to keep some sort of test for Saml2SecurityTokenHandler, but it doesn't have any other public properties (aside from Serializer) that are checked by IdentityComparer. 

I'll go ahead and remove it entirely.